### DISCUSSION TITLE
html reports deprecated soon?

### DISCUSSION THREAD

**Discussion by SmartALB** [artilleryio/artillery#2405] [Artillery]
On https://www.artillery.io/docs/reference/cli/report I read that the report command will soon be deprecated. What exactly does that mean? - Does it mean it will not be possible at all to generate HTML reports, and the only alternative will be the upcoming dashboard?

**Comment by hassy** [comment]
Hi @SmartALB, yes indeed. Reporting in the Artillery dashboard is a huge upgrade compared to the HTML reports generated by the CLI. Those reports will be available on the free plan.

**Reply by SmartALB** [reply]
I am happy that this will also be available in the free plan.
Is it already known when the dashboard will be approximately available?
